src/mcts/search.cc

```diff
@@ -467,7 +496,7 @@ void Search::SendMovesStats() const REQUIRES(counters_mutex_) {
     LOGFILE << "=== Move stats:";
     for (const auto& line : move_stats) LOGFILE << line;
   }
-  for (auto edge : root_node_->Edges()) {
+  for (auto& edge : root_node_->Edges()) {
```

Comment:
Hmmm it may make sense to make iterators return pointers rather than references to avoid such mistakes.